### PR TITLE
doc/man/cephfs-top.rst: add missing options: --delay, --conffile

### PR DESCRIPTION
**Pull request by wxypro**
add missing options: --delay, --conffile

Signed-off-by: wangxinyu <wangxinyu@inspur.com>

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [ ] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by lxbsz — APPROVED** [review]

File: doc/man/8/cephfs-top.rst [hunk 1]
```diff
@@ -36,6 +36,14 @@ Options
...
    Perform a selftest. This mode performs a sanity check of ``stats`` module.
 
+.. option:: --conffile [CONFFILE]
+
+   Path to cluster configuration file
+
+.. option:: -d [DELAY], --delay [DELAY]
+
+   Interval to refresh data (default: 1)
+
```
Comment (anthonyeleven) [inline doc/man/8/cephfs-top.rst:46]: Specify units - Seconds? Minutes? Microfortnights?

**Review by joscollin — APPROVED** [review]

**Review by kotreshhr — APPROVED** [review]

File: doc/man/8/cephfs-top.rst [hunk 2]
```diff
@@ -36,6 +36,14 @@ Options
 
    Perform a selftest. This mode performs a sanity check of ``stats`` module.
 
+.. option:: --conffile [CONFFILE]
+
+   Path to cluster configuration file
+
+.. option:: -d [DELAY], --delay [DELAY]
+
+   Interval (in seconds) to refresh data (default: 1 second)
```
Comment (nmshelke) [inline doc/man/8/cephfs-top.rst:45]: Just suggestion:
`Data refresh interval in seconds (default: 1 second)`

File: src/tools/cephfs/top/cephfs-top [hunk 3]
```diff
@@ -601,7 +601,7 @@ if __name__ == '__main__':
     parser.add_argument('--selftest', dest='selftest', action='store_true',
                         help='Run in selftest mode')
     parser.add_argument('-d', '--delay', nargs='?', default=DEFAULT_REFRESH_INTERVAL,
-                        type=float_greater_than, help='Interval to refresh data '
+                        type=float_greater_than, help='Interval (in seconds) to refresh data '
```
Comment (nmshelke) [inline src/tools/cephfs/top/cephfs-top:604]: Just suggestion:
`Data refresh interval in seconds (default: 1 second)`

**Review by nmshelke — APPROVED** [review]
Data

**Comment by vshankar** [comment]
jenkins retest this please

**Comment by vshankar** [comment]
@wxypro could you please and push?

**Comment by joscollin** [comment]
> @wxypro could you please and push?

"rebase and push". This is to resolve `docs/readthedocs` jenkins build failure.

File: src/tools/cephfs/top/cephfs-top [hunk 4]
```diff
@@ -601,7 +601,7 @@ if __name__ == '__main__':
     parser.add_argument('--selftest', dest='selftest', action='store_true',
                         help='Run in selftest mode')
     parser.add_argument('-d', '--delay', nargs='?', default=DEFAULT_REFRESH_INTERVAL,
-                        type=float_greater_than, help='Interval to refresh data '
+                        type=float_greater_than, help='Data refresh interval in seconds '
```
Comment (vshankar) [inline src/tools/cephfs/top/cephfs-top:604]: "Metric refresh interval in seconds" sounds much more appropriate.

**Review by vshankar — COMMENTED** [review]
Minor nit, otherwise looks good.

File: doc/man/8/cephfs-top.rst (hunk 1 above)
Comment (anthonyeleven) [reply doc/man/8/cephfs-top.rst:46]: The second second is a bit redundant but this works.

**Comment by anthonyeleven** [comment]
jenkins test docs

**Comment by anthonyeleven** [comment]
jenkins render docs

**Comment by vshankar** [comment]
jenkins retest this please

**Comment by vshankar** [comment]
@wxypro please rebase

**Review by rishabh-d-dave — APPROVED** [review]

**Comment by rishabh-d-dave** [comment]
@wxypro Can you please rebase the PR branch? Can't merge without it and I don't have permissions to update the PR branch.

**Comment by rishabh-d-dave** [comment]
Rebased version  is posted here #47139. Closing in favour of it.

**Pull request closed without merging**